charts/pelorus/charts/exporters/templates/_deploymentconfig.yaml

```diff
@@ -29,17 +29,13 @@ spec:
       - name: {{ .app_name }}
         imagePullPolicy: Always
         envFrom:
-      {{- if .env_from_configmaps }}
-        {{- range .env_from_configmaps }}
+      {{- range .env_from_configmaps | default (list "default-pelorus-config" "default-deploytime-config") }}
         - configMapRef:
             name: {{ . }}
-        {{- end}}
       {{- end}}
-      {{- if .env_from_secrets }}
-        {{- range .env_from_secrets }}
+      {{- range .env_from_secrets }}
```

Comment:
Go Templates inherit Go's semantics for arrays: the "zero" / nil value is considered an empty array.